packages/table/src/headers/columnHeader.tsx

```diff
@@ -77,7 +77,7 @@ export class ColumnHeader extends React.Component<IColumnHeaderProps, {}> {
 
     public render() {
         const { grid, viewportRect, columnIndexStart, columnIndexEnd } = this.props;
-        const cells: React.ReactElement<any>[] = [];
+        const cells: Array<React.ReactElement<any>> = [];
```

Comment:
ok gotcha. that makes more sense: if you're already dealing with generics, use `Array` the same way.